src/main/java/de/tum/in/test/api/ast/model/JavaFile.java

```diff
@@ -47,12 +60,12 @@ public CompilationUnit getJavaFileAST() {
 	 * @return The information of the Java-file packed into a JavaFile object (null
 	 *         if the file is not a Java-file)
 	 */
-	public static JavaFile convertFromFile(Path pathOfFile) {
+	public static JavaFile convertFromFile(Path pathOfFile, Boolean excludeMainMethod) {
```

Comment:
Why do you use a boxed `Boolean` here?

Reply:
Should be boolean, but will change this according to Christian's comments